src/components/Input/Input.md

```diff
@@ -1,11 +1,19 @@
-### Default Style
+Text inputs are the basic building blocks forms. They allow users to enter various types of data into web-based forms.
```

Comment:
Not sure what you're trying to say here, but "Text inputs are the basic building blocks forms" does not make sense to me.

Reply:
These intro statement are all verbatim from the Rivet docs.

Reply:
Well, there you go.

Reply:
This seems like a case where we should deviate from the Rivet docs and write something that makes sense and file an issue with Rivet to fix the documentation there and then rectify it if/when it gets fixed.